lib/common/tests/schemas/crm_schema_init_test.c

```diff
@@ -141,11 +141,8 @@ extra_schema_files(void **state)
     assert_schema("pacemaker-3.1", 15);
     assert_schema("pacemaker-3.2", 16);
 
-    // @COMPAT pacemaker-next is deprecated since 2.1.5
-    assert_schema("pacemaker-next", 17);
-
     // @COMPAT none is deprecated since 2.1.8
-    assert_schema(PCMK_VALUE_NONE, 18);
+    assert_schema(PCMK_VALUE_NONE, 17);
```

Comment:
> Not sure why this even works, since we have schemas after `"pacemaker-3.2"` and even have a `"pacemaker-4.0"`. The `setup()` function isn't shedding light on it for me.

See tests/schemas/Makefile.am. We create a test directory with a subset of schemas for regression testing purposes.

> This file should be renamed to `pcmk__schema_init_test.c`

Thanks